server/src/main/java/io/crate/expression/scalar/array/ArrayArgumentValidators.java

```diff
@@ -58,4 +58,15 @@ public static void ensureSingleArgumentArrayInnerTypeIsNotUndefined(List<DataTyp
                 "When used with only one argument, the inner type of the array argument cannot be undefined");
         }
     }
+
+    public static void ensureArrayArgAndObjectArgAreNotBothUndefined(List<DataType<?>> dataTypes, String functionName) {
```

Comment:
Is this now unused? If so can we remove it?

Reply:
yes sure, removed!